From 60d95eb1b308f0173cc2621c6ae144f38514b8b1 Mon Sep 17 00:00:00 2001
From: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Date: Tue, 26 Mar 2019 11:26:28 -0700
Subject: [PATCH 0579/1852] v4l: xilinx: vip: Use correct media bus code for
 RGB 10bit format

commit 4dae24a483a6224199716bbe4829b14d0f473e61 from
https://github.com/Xilinx/linux-xlnx.git

This patch uses the correct media bus code for RGB 10bit pixel format.

Signed-off-by: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vip.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vip.c b/drivers/media/platform/xilinx/xilinx-vip.c
index 15a9942811d6..57c06d499f77 100644
--- a/drivers/media/platform/xilinx/xilinx-vip.c
+++ b/drivers/media/platform/xilinx/xilinx-vip.c
@@ -74,7 +74,7 @@ static const struct xvip_video_format xvip_video_formats[] = {
 	  4, 32, V4L2_PIX_FMT_BGRX32, 1, 1, 1, 1 },
 	{ XVIP_VF_XRGB, 8, NULL, MEDIA_BUS_FMT_RBG888_1X24,
 	  4, 32, V4L2_PIX_FMT_XBGR32, 1, 1, 1, 1 },
-	{ XVIP_VF_XBGR, 10, NULL, MEDIA_BUS_FMT_RBG888_1X24,
+	{ XVIP_VF_XBGR, 10, NULL, MEDIA_BUS_FMT_RBG101010_1X30,
 	  4, 32, V4L2_PIX_FMT_XBGR30, 1, 1, 1, 1 },
 	{ XVIP_VF_XBGR, 12, NULL, MEDIA_BUS_FMT_RBG121212_1X36,
 	  3, 40, V4L2_PIX_FMT_XBGR40, 1, 1, 1, 1 },
-- 
2.31.1

